src/guiBuilder.js

```diff
@@ -0,0 +1,35 @@
+/* globals window */
+
+// TODO : Re-evaluate the api of the gui-builder
+var GUIBuilder = function(buttons) {
+  this.buttons = [];
+  this.group = window.game.add.group();
+
+  for (var i = 0; i < buttons.length; i++) {
+    var btn = buttons[i];
+    this.buttons[i] = this.makeButton(btn.x, btn.y, btn.name, btn.size, btn.anchor, btn.callback);
+    this.group.add(this.buttons[i]);
+  }
+};
+
+GUIBuilder.prototype = {
+  // makeButton is used in the constructor of GUIBuilder, but can
+  // also be used on it's own
+  makeButton: function(x, y, name, size, anchor, callback) {
+    if (!size) { size = 25; }
+    if (!anchor) { anchor = [0.5, 0.5]; }
+
+    var button = window.game.add.text(x, y, name, {resolution: window.devicePixelRatio});
```

Comment:
That was my ultimate goal but I wanted you guys to get a look at this before then, I'm hoping we can work on the api together before we make a more extensible function. I also wanted to keep this function simple so that we don't get to crazy with weird buttons and stuff.